setup.cfg

```diff
@@ -0,0 +1,31 @@
+[pycodestyle]
+max-line-length = 99
+
+[pylint.MASTER]
+jobs = 3
+max-line-length = 99
+fail-under=9.9
```

Comment:
I know why you did this but I think we should reduce the number of errors to 0 instead. The main reason is that, if the new error appears in the result, it can be skipped because it won't be fully visible.

I would even say that if resolving them is problematic, I would skip them locally (at the line where they appear). It will be much better solution than seeing them all the time 😉